atd-vze/src/views/Crashes/Recommendations.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from "react";
+import { useQuery, useMutation } from "@apollo/react-hooks";
+import { Card, CardHeader, CardBody, CardFooter, Table, Button } from "reactstrap";
+import { recommendationsDataMap } from "./recommendationsDataMap";
+import { GET_RECOMMENDATIONS } from "../../queries/recommendations";
+import { useAuth0, isReadOnly } from "../../auth/authContext";
+
+// declare fatality review board recommendations component
+const Recommendations = ({ crashId }) => {
+
+  // fetch data from database using graphQL query
+  const { loading, error, data, refetch } = useQuery(GET_RECOMMENDATIONS, {
+    variables: { crashId },
+  });
+
+  if (loading) return "Loading...";
```

Comment:
do y'all use a loading spinner elsewhere in VZ? would be a nice touch to add one.